tests/internals/scaled_object_validation/scaled_object_validation_test.go

```diff
@@ -189,6 +215,20 @@ func testScaledWorkloadByOtherHpa(t *testing.T, data templateData) {
 	KubectlDeleteWithTemplate(t, data, "hpaTemplate", hpaTemplate)
 }
 
+func testScaledWorkloadByOtherHpaWithOwnershipTransfer(t *testing.T, data templateData) {
+	t.Log("--- already scaled workload by other hpa ownership transfer ---")
+
+	data.HpaName = hpaName
+	err := KubectlApplyWithErrors(t, data, "hpaTemplate", hpaTemplate)
+	assert.NoErrorf(t, err, "cannot deploy the hpa - %s", err)
+
+	data.ScaledObjectName = scaledObject1Name
+	err = KubectlApplyWithErrors(t, data, "scaledObjectTemplate", ownershipTransferScaledObjectTemplate)
+	assert.NoErrorf(t, err, "can deploy the scaledObject - %s", err)
+
+	KubectlDeleteWithTemplate(t, data, "hpaTemplate", hpaTemplate)
+}
```

Comment:
Could you create another HPA for this specific e2e test case? If we share it, tests can affect to others